README.md

```diff
@@ -18,8 +18,30 @@ configuration and creating a Python entry point that launches the app.
 
 ## Development Install
 
-```
+``` shell
 git clone https://github.com/jupyterlab/jupyterlab_server.git
 cd jupyterlab_server
 pip install -e .
 ```
+
+## Testing
+
+To create a local test setup run the following commands:
+
+``` shell
+git clone https://github.com/jupyterlab/jupyterlab_server.git
+cd jupyterlab_server
+python3 -m venv venv
+source venv/bin/activate
```

Comment:
Since different devs could be using different setups, maybe the environment handling could be left for the user. There are many other tools (like conda) so I would rather not provide these steps, but only tell the user how to install the test dependencies with some info on using environments, defered to the specific tools (pipenv, venv condac etc.)

Reply:
OK, thanks for the feedback! I will leave it neutral in terms of virtual environment options and only mention that one probably should use a tool for that.

Reply:
👍